### ISSUE TITLE
CPU Usage is not accurate on hosts

### ISSUE THREAD

**Issue by paulbellamy** [weaveworks/scope#1664]
It's being reported out of 100%, on hosts, not num CPUs.

**Comment by rade** [comment]
AFAICT from looking at the code we report an overall cpu usage, i.e. accumulated over all cores, so that 100% corresponds to "all cores busy". But we report the limit as num_cpus * 100%. So this looks wrong - we should either report num_cpu * value, or have the limit as 1*100%.

A quick experiment on my 4 core machine does show the discrepancy - if I get my machine to max out all its CPUs, I see 100% reported in the details panel, but the squiggly line is nowhere near the max.

But since the fix would appear to be simply removing the num_cpu multiplier on the limit or adding it to the value - I've tried that, and the graph looks fine then - I wonder what I am missing, @paulbellamy?

As an aside, I've noticed that container CPU stats report 100% when one core is used, and we set the limit to num_cores*100%. So that would be consistent with multiplying the host cpu value by num_cores. Is there a compelling reason for doing it this way rather than always set the limit to 100%, with a single maxed out core being reported as 100%/num_cores? I suppose the limit provides an indirect indication of the number of cores. But it looks kinda weird to use >100% of a resource.
